pkg/platform/runtime/buildscript/file.go

```diff
@@ -43,11 +43,15 @@ func UpdateOrCreate(dir string, newScript *model.BuildScript) error {
 		return nil
 	}
 
+	script, err = NewScriptFromBuildExpression([]byte(newScript.String()))
```

Comment:
nit: script could also have a `Bytes()` function since it seems like you're only ever converting the string to a slice of bytes

Reply:
When David reviewed my Build Script PR, he suggested the `String()` method because it's quite a common API convention, so I'm going with that :)